.github/workflows/main.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           conda env list
           conda list
 
+      - name: Lint
```

Comment:
This can maybe be removed? I guess the pre-commit CI thing runs it.

Reply:
I think the pre-commit CI will run without this.